plugins/ps/subcommands/restore

```diff
@@ -7,11 +7,20 @@ source "$PLUGIN_AVAILABLE_PATH/ps/functions"
 ps_restore_cmd() {
   declare desc="starts all apps with DOKKU_APP_RESTORE not set to 0 via command line"
   local cmd="ps:restore"
+
+  if which parallel > /dev/null 2>&1; then
+    dokku_apps | parallel dokku ps:start
+    return
```

Comment:
you don't seem to check DOKKU_APP_RESTORE here for each app like it's done below

Reply:
@josegonzalez one year later, here a PR that fixes it for current master #3023